src/Symfony/Component/String/Tests/Inflector/EnglishInflectorTest.php

```diff
@@ -298,6 +298,7 @@ public static function pluralizeProvider()
             ['waltz', 'waltzes'],
             ['wife', 'wives'],
             ['icon', 'icons'],
+            ['hippocampus', 'hippocampuses'],
```

Comment:
Strange that you've picked this example because the plural of "hippocampus" is actually "hippocampi".

https://www.merriam-webster.com/dictionary/hippocampus

Reply:
well, both are accepted apparently :)
latin roots...

Reply:
Should I change it anyways ?

Reply:
> well, both are accepted apparently :) latin roots...

Is that so? I could only find sources that list "hippocampi" that the only plural. Also, "hippocampuses" causes pain in my hippocampus. 😬 Sorry. 😓 

Anyway, "hippocampus" is currently pluralized to "hippocampi" which is correct and I think we should not change it. Regarding the your initial issue, "campus" should be pluralized to "campuses". Can you fix that and leave "hippocampus" intact.